benchmarks/nel/scripts/compile_corpora.py

```diff
@@ -12,4 +12,5 @@ def main(dataset_name: str):
 
 
 if __name__ == "__main__":
-    typer.run(main)
+    # typer.run(main)
+    main("mewsli_9")
```

Comment:
Is this intentional, or is it a result from testing?

Reply:
The latter.